### ISSUE TITLE
Research Intake: New meta-study on partner & housemate transmission

### ISSUE THREAD

**Issue by blanchardjeremy** [microcovid/microcovid#606]
https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2774102

Takeaways:
* Interesting. It suggests adult household transmission to partners is 28% and non-partners is 17%. Might be slightly higher for symptomatic cases.
* They don't differentiate between asymptomatic and presymptomatic.

**Comment by beshaya** [comment]
@blanchardjeremy I think the numbers you cite are transmission from (adults) to (adults or children)

>The estimated mean household secondary attack rate was significantly higher to adult contacts (28.3%; 95% CI, 20.2%-37.1%) than to child contacts (16.8%; 95% CI, 12.3%-21.7%; P < .001)

I think "to adult contacts" in the above includes spouses.

>  Estimated mean household secondary attack rate to spouses (37.8%; 95% CI, 25.8%-50.5%) was significantly higher than to other contacts (17.8%; 95% CI, 11.7%-24.8%).

"To other contacts" includes children.

Children might be half as likely to transmit, but the data isn't strong enough to say this with much confidence
>Estimated mean household secondary attack rate from adults (15.2%; 95% CI, 6.2%-27.4%) was not significantly different than that from children (7.9%; 95% CI, 1.7%-16.8%). 

Also, there's good evidence that a blanket household transmission rate isn't particularly accurate, since contact frequency matters a lot:
> Contact frequency with the index case was associated with higher odds of infection, specifically
at least 5 contacts during 2 days before the index case was confirmed,70 at least 4 contacts and 1 to
3 contacts,63 or frequent contact within 1 meter.

Households of 4 or more were associated with less transmission than households of 2. I hypothesize this is largely explainable by spouse vs child (although the SAR for households of 2 is higher than the SAR for spouses!)
> Estimated mean household secondary attack rate for households with 1 contact (41.5%; 95% CI, 31.7%-51.7%) was significantly higher than households with at least 3 contacts (22.8%; 95% CI, 13.6%-33.5%; P < .001) but not different than households with 2 contacts (38.6%; 95% CI, 17.9%-61.6%).
![image](https://user-images.githubusercontent.com/5877421/103820629-8d517200-5021-11eb-90e0-104b950af5bf.png)

## Summary
This suggests partner transmission rate of 38% (vs 48% in current model). It's unclear what the adult-to-non-spouse-adult SAR is from this paper; using the original 1.6x multiplier from Li et al. would suggest 23% (although there's no particular to put so much weight on Li et al).

HOWEVER - I think one could compute this from the data directly, using the charts on pages 7 and 8.

**Comment by beshaya** [comment]
Li et al. should especially not be given extra weight because it is included in the meta analysis :)

**Comment by blanchardjeremy** [comment]
(FYI, my original post was a copy-paste from Slack or our FB thread, not my takeaways. I should've said that I was quoting others.)

@beshaya Thanks so much for doing the in-take! Looks great. Would you recommend we adopt that partner number?

Also, how do these numbers computer with the B117 COVID variant? We'd still need to 1.5x these numbers, yeah?

**Comment by beshaya** [comment]
yeah, 1.56x for b117. My guess is it will take a few months before we get B117 data with similar fidelity to this paper. That brings the B117 spouse number to 59%, still higher than it is today.

I'd recommend that partner number. Our current number is "a highly speculative number" and "the best we have right now" ~https://www.microcovid.org/paper/14-research-sources

**Comment by blanchardjeremy** [comment]
@beshaya Great. That sits well with me. (Though others should take a look too). Are you recommending a change to housemate as well or no?

**Comment by catherio** [comment]
The housemate number would be 17% * 1.5 = 25.5%. I think we should leave it at the 30% value because it's close enough, within "might be slightly higher for symptomatic cases", and it's very load-bearing in pod people's usage so that changing it would be disruptive.

**Comment by catherio** [comment]
(I also hadn't seen Koh et al: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0240205; not sure if we need to do anything about it, just offering it)

"The pooled household SAR was 18.1%"; relative risk for spouses is 2.39, which seems to me to make 43%.

**Comment by beshaya** [comment]
@catherio that high level result is pooling everything - adults, children, and spouses. The 2.39 factor is between spouses and non-spouses, not between spouses and pooled average. The number is explicitly written out in the paper:
> "In all seven studies, the SAR to spouses (37.5%; 95% CI: 22.2%, 52.7%) was higher than to other household contacts (16.3%; 95% CI: 10.6%, 22.1%) 

Also, if we intend out numbers to be adult to adult with a discount for children, it is important not to remove children from the base number:
> overall, the household SAR in adults (33.3%; 95% CI: 24.4%, 42.1%) was significantly higher than that in children (16.9%; 95% CI: 10.9%, 22.9%) (Fig 6).

For spouses:

Again, this does not break down "adult-to-non-spouse-adult"

So to summarize:
|Metric                                        |Madwell et al.|Koh et al.|
|------------------------------|--------------|---------|
|Overall                                        | 17%               | 18.1%     |
|Spouse to spouse                     | 37.8%            | 37.5%    |
|X to adults (inc spouses)         | 28.3%            | 33.3%   |
|X to Children                             | 16.8%             | 16.9%   |
|Children to X.                            | 7.9%               |  ???       |
|Adults to X.                                | 15.9%            |   ???      |
|Adult to adult (not spouse).     | ???                 |  ???       |

I don't ascribe too much confidence to the similarity of these numbers, since I suspect that they're actually pulling the same papers to meta-analyze.

But they both suggest 38% pre-VOC 202012/01 partner-to-partner risk.

**Comment by beshaya** [comment]
Ok here's my analysis of adult to non-spouse adult.

Of the 7 papers that break down spouse/other, Sun et al, Xin et al, Liu et al, Wu et al, Li et al, Chaw et al, Lewis et al, the first 3 do not include child/adult breakdown. The next 2 have different numbers of participants for adult/child and spouse/other, so I cannot determine how many non-spouse adults there were. However, I was able to pull Li et al and see they directly break down "Not spouse (excluded children)"

Chaw et al is an outlier for studies that break down spouse/non spouse (6% compared to 18% average), so I chose not to include it given that it would be 1 of 2 studies.

For Lewis et al. I assumed that all spouses are adults, so subtracting the children from the non-spouses, I calculated a 25% chance of adult-to-non-spouse-adult transmission chance. Lewis et al found an overall 33% SAR for partners; scaling to the overall 38% SAR for partners would give us a 28% adult-to-non-spouse-adult SAR.

For Li et al, we can directly read off a 28% spouse SAR and a 17% non-spouse SAR. Scaling this to 38% Spouse SAR gives a 23% adult-to-non-spouse SAR.

To combine these two, I used the weights assigned by Madewell et al, which assigned slightly higher weight to Li et al. across the board. This comes to a final **adult-to-non-spouse-adult SAR of 26%**

You can see my work here: https://docs.google.com/spreadsheets/d/1JvnNIXuDTMOtxq1XPXZB60T6u3rDZ-B8nmmGD9kbhro/edit#gid=0

**Comment by beshaya** [comment]
@catherio for approval

**Comment by catherio** [comment]
Summarizing for myself & to check my understanding:
* Madewell and Koh agree X-to-adult transmission is in the 30% ballpark
* The overall numbers are lower (<20%) and this is probably because it includes X->kids and kids->X which are both noticeably lower
* Ben was able to pull specific subgroup numbers from Chaw, Lewis, and Li that are detailed enough to give us a separate non-spouse adult number, which is 60-80% lower than the spouse numer.
* We suggest spouse-to-spouse at ~38%, and spouse-to-non-spouse-adult at 70% that value which is 26%

**Comment by catherio** [comment]
Looks like after B117, these will adjust to the ballpark of 57% and 40% (from current 50% and 30%).

That feels/sounds right to me.

Let's think carefully about rollout to pod/spreadsheet users to avoid unthoughtful disruption to house policies that depend on the 30% number. @blanchardjeremy 

but epistemically I am 👍 to this

**Comment by catherio** [comment]
I almost wonder if it's best to do "partner/spouse who you CAN/WILL isolate from if they get sick" as separate from "partner/spouse who you CAN'T/WON'T isolate from if they get sick" 🧐

**Comment by beshaya** [comment]
Oh that's a really great question. It should make a huge difference; especially to partners who live separately. I just have no idea what that value should be! Do you have any thoughts?

**Comment by catherio** [comment]
No idea yet! Sounds like worth opening a new issue for!

**Comment by catherio** [comment]
btw one of the coauthors on Curmei et al says "My main question is if they correct for false negatives and for the fact that some studies only tested symptomatic cases. I ctrl+f'd for both terms and didn't see it" => soooo, this might be worth looking into before approving lower numbers. I'm re-marking this as needing work just to at least look into what Curmei et al did there and what we think it means for this study.

**Comment by beshaya** [comment]
Good catch. Is this also an issue with the studies that the original
numbers are from?

**Comment by blanchardjeremy** [comment]
Just to check: we're still waiting on this last question, yeah?

**Comment by beshaya** [comment]
Test strategies are summarized in eTable 4 of the supplement: https://cdn.jamanetwork.com/ama/content_public/journal/jamanetworkopen/938592/zoi200987supp1_prod_1607354087.12032.pdf

The surveyed studies used a mix of symptomatic and symptomatic + non-symptomatic strategies, along with varied numbers of tests.

Both Lewis et al and Li et al tested both symptomatic + asymptomatic once, and did subsequent tests for symptomatic people. They monitored for symptoms over 14 days.

Both Lewis and Li both found overall SARs that were lower than the meta-study average; the proposed numbers are 15% and 30% higher than what was found in those studies, so including the rest of the data brings the numbers up. 

I don't have the expertise (or free time) needed to re-meta analyze the 54 studies or assess whether Madewell et al properly weighted for varied testing strategy though. If this point matters to you, I'd suggest hiring someone (Elizabeth van Nostrand?) to go through every study and decide if they should be discarded.

**Comment by catherio** [comment]
note to self that Chana wanted me to look at https://www.thelancet.com/pdfs/journals/laninf/PIIS1473-3099(20)30471-0.pdf

**Comment by catherio** [comment]
> I don't have the expertise (or free time) needed

Well, a cheaper strategy is just to compare to Curmei et al: "The pooled central estimate of the SAR is 30%  [...] If results are not corrected for false negatives and asymptomatics, the pooled central estimate for the SAR is 20%"

This suggests that the Madewell and Koh numbers in the ~17-18% ballpark are actually quite concordant with Curmei et al. if we assume the underlying studies had the same distribution of testing strategies.

**Comment by catherio** [comment]
I also just want to zoom out and ask, though - what is the priority in trying to beat Curmei et al. at this epistemics game? It seems hard to do. The bar of being better at this game than Curmei et al. seems really high, and I'd be inclined not to try (just as I didn't try to beat ~OSHA at the P100 estimation game, like we're not rolling our own prevalence anymore now that we don't have to, etc.)

**Comment by catherio** [comment]
This is definitely the time to squint and shoot from the hip according to what you really believe holistically (having seen everything you've seen), and just explain in the white paper why you did it that way. 

@beshaya is the one eyeballs-deep in this, so Ben if you just tell me what you really think the numbers should be (after seeing that Curmei et al had to adjust up by +50% to factor in the testing strategies), I will probably approve it.

**Comment by beshaya** [comment]
Ok, I think what you're saying is that Curmei et al compensate for under testing / false negatives, which means they actually agree with Madewell et al.  The final implication, then is that Curmei et al's numbers are correct _for D614G (Wuhan variant)_, which means that, for B117 we need to use **.45 / .72 for housemate/partner multipliers?**

Is that correct?
